doc/integrator/extend_application.rst

```diff
@@ -33,24 +145,186 @@ have an attribute slot. The slot says where the component should be added:
 - ``tool-<panel-name>`` -> in the tools panel on the right of the map, when the tool is activated.
 - ``footer-<panel-name>`` -> in the footer part of the page, when the panel is activated.
 
-In the ``vars.yaml`` file your interface should be declared like that:
 
-.. code:: yaml
+Create a new WebComponent
+-------------------------
 
-   interfaces:
-     - name: desktop
-       type: canvas
-       default: true
+In this tutorial we will create a new WebComponent based on `Lit <https://lit.dev/>`_,
+and build by `Vite <https://vitejs.dev/>`_.
 
-Custom docker-compose service
------------------------------
+We will add a button in the tools bar, that open a new tool panel, that can be used to send a feedback.
```

Comment:
Suggestion:
"We will add a button in the tools bar which opens a new tool panel and that can be used to send a feedback."